telemetry/specs/INT.mdk

```diff
@@ -1009,24 +1006,28 @@ each bit corresponds to a specific standard metadata as specified in Section 3.
   Details of the metadata semantics YANG model can be accessed at the link below:
   https://github.com/p4lang/p4-applications/blob/master/telemetry/code/models/p4-dtel-metadata-semantics.yang
   
-  Bits 0 - 13 are Baseline INT Instructions and Bit 14, Domain Specific Instruction.
-  Domain Specific Instruction is an instruction that requires additional 
-  processing of Domain Specific ID, Domain Specific Flags (DS Flags) and 
-  Domain Specific Field (DS Field). If the Domain Specific ID doesn't match the 
-  Domain ID of the node, then the transit node is required to pad the node's 
-  INT Metadata stack with the special all-ones reserved value for Domain Specific 
+  Bits 0 - 14 are Baseline INT Instructions.
+  
+  * Domain Specific ID (16b): the unique ID of the INT Domain.
+  
+  * DS Instruction (16b): Instruction bit map specific to the INT domain identified by the 
+  Domain Specific ID. Domain Specific Instruction is an instruction that requires additional 
+  processing of Domain Specific Flags (DS Flags) for the INT Domain identified by Domain Specific ID. 
+  If the Domain Specific ID doesn't match the Domain ID of the node, then the transit node is 
+  required to pad the node's INT Metadata stack with the special all-ones reserved value for Domain Specific 
   Metadata length as calculated by subtracting the Baseline Metadata Length from Hop_ML.
   
   If the Domain Specific ID matches the node's Domain ID then additional processing 
```

Comment:
I believe a node could have knowledge of more than one domain ID. That is why I suggested:
s/matches the node's Domain ID/matches any Domain ID known to this node

Reply:
Good Point. Fixed it.